index.js

```diff
@@ -1,12 +1,22 @@
 var fs = require('fs');
 var path = require('path');
+var writeFile = require('broccoli-file-creator');
+var mergeTrees = require('broccoli-merge-trees');
 
 module.exports = {
   name: 'ember-waypoints',
 
-  contentFor: function(type) {
-    if (type === 'vendor-suffix') {
-      return 'if (typeof(document) !== "undefined") {\n' + fs.readFileSync(path.join(this.project.bowerDirectory + '/jquery-waypoints/waypoints.js')) + '\n}';
-    }
+  treeForVendor(tree) {
+    var content = `if (typeof(document) !== "undefined") {
+      ${fs.readFileSync(path.join(`${this.project.bowerDirectory}/jquery-waypoints/waypoints.js`))}
+    }`;
+
+    var waypointsTree = writeFile('/waypoints.js', content);
+
+    return mergeTrees([tree, waypointsTree]);
```

Comment:
I suspect this isn't necessary, I might be able to just return `[waypointsTree]` instead